test/math.js

```diff
@@ -439,6 +439,14 @@ describe('Math', function () {
       expect(Object.is(-0, Math.round(-0.5 + Number.EPSILON / 3.99))).to.equal(true);
       expect(Object.is(-0, Math.round(0 - Number.EPSILON / 3.99))).to.equal(true);
     });
+
+    it('returns 1 / Number.EPSILON + 1 for 1 / Number.EPSILON + 1', function () {
+      expect(Math.round(1 / Number.EPSILON + 1)).to.equal(1 / Number.EPSILON + 1);
```

Comment:
Can you elaborate a bit more on what current browsers do for these inputs, and what (if anything) the spec requires for it?